modules/islandora_scopus/islandora_scopus.module

```diff
@@ -61,7 +61,7 @@ function islandora_scopus_block_view($delta = '') {
               'apiKey' => $api_key,
               'httpAccept' => 'text/html',
             ]);
-            $scopus_url = "http://api.elsevier.com:80/content/abstract/citation-count?{$data}";
+            $scopus_url = "https://api.elsevier.com/content/abstract/citation-count?{$data}";
```

Comment:
Good practices tell me this URL should be exposed to the user to be modified anytime its needed. The idea of forcing people to update by a certain date because our hardcoded URL will de deprecated should be avoided in the future.

Reply:
@DiegoPino I understand that feeling, but alternatively, its really on Elsevier to keep a stable API up. Forcing people to update because they will deprecate their own API is a massive fail on their part, not ours, and allowing people to alter the URL (variable in an admin menu?) introduces some weird complexity where people might have issues because they have set the wrong URL for the API and the module no longer works.

In a nutshell: I think API URLs *should* be hardcoded, because API URLs shouldn't change. People are gonna have to update badges because of Elsevier's fail anyways, lets not normalize their bad behavior.

^ Strong opinion held loosely, just playing devil's advocate here and opening up a debate on the idea.

Reply:
@bryjbrownI totally get your point and I appreciate the role of devil's advocate here + your experience (IR archangel!) on this. The reason I went for this (Issue I saw) is that our folks won't blame Elsevier when stuff stop working because we are the ones providing the UI and the connection and forcing updates is something we have done only for security reasons. A bit again about the role of Islandora in this whole scenario where we rely more and more on external data providers, probably this will play against integrators quite more in the future. There are even some API providers that version their APIs and deprecate old endpoints like /myendpoint/v1/? etc... So, now i have mixed feelings. Like, is fedora an API? and why are we asking people to add their fedora endpoints? I propose we discuss this tomorrow! Worst case for @bondjimbond (sorry man) you would have to rollback a bit to where I started interacting. Probably related to my other comment on how we allow stable fallbacks and permit resetting forms to factory defaults. Thanks @bryjbrown, this is a good discussion!

Reply:
@DiegoPino If we have to roll back, then it's just a matter of going back to the first commit to this branch. No biggie.